src/test/groovy/graphql/GraphQLErrorTest.groovy

```diff
@@ -31,7 +31,7 @@ class GraphQLErrorTest extends Specification {
                 .description("Test ValidationError")
                 .build()                                                                               |
                 [
-                        locations: [[line: 666, column: 999], [line: 333, column: 0]],
+                        locations: [[line: 666, column: 999], [line: 333, column: 1]],
```

Comment:
Why do these numbers chance - what caused that?

Reply:
In the past, our tests had "0" being used as a valid column location. This PR bans anything less than 1 for line and column number, which made these old tests fail. So I adjusted these old tests to use a valid location number

Then added a new test with invalid numbers to check the new filtering behaviour

Reply:
The source locations for these old tests are generated by a method lower down in this file
```
List<SourceLocation> mkLocations()
```